WcaOnRails/app/views/static_pages/education.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, t('education.title')) %>
+
+<div class="container">
+  <h1><%= yield(:title) %></h1>
+
+  <p>
+    Download the <%= link_to t('education.competitor_tutorial'), "/edudoc/competitor-tutorial/tutorial.pdf" %>.
```

Comment:
Shouldn't "download the" also be translated?

Reply:
Yes definitely, but I'm not even sure a page is needed so I'll loop in the WQAC first.

Reply:
Done now that @leonopulos has replied in the email thread.